src/aktualizr_secondary/secondary_tcp_server.h

```diff
@@ -14,11 +14,14 @@ class SecondaryInterface;
  */
 class SecondaryTcpServer {
  public:
-  SecondaryTcpServer(Uptane::SecondaryInterface& secondary, const std::string& primary_ip, in_port_t primary_port,
-                     in_port_t port = 0);
+  enum class ExitReason {
+    kNA,
```

Comment:
"NA" for "not applicable"? Minor, but maybe worth spelling out.

Reply:
I've also renamed Other to Unknown, more consistent with other enums we have.